src/_internal/logger.ts

```diff
@@ -0,0 +1,29 @@
+import { homedir } from "os"
+import { join } from "path"
+import { existsSync, mkdirSync, appendFileSync } from "fs"
+
+const LOG_DIR = join(homedir(), ".next-devtools-mcp")
```

Comment:
should we add the log folder under the package? like `node_modules/next-devtools-mcp/.telemetry` rather than sharing in the same log file for all mcp instance. also you don't need to delete them manually